configs/samples/cel_pgsql.conf.sample

```diff
@@ -3,71 +3,70 @@
 ;
 
 ; Sample Asterisk config file for CEL logging to PostgreSQL
-;
+
 ; CEL field names:
-;
-;	eventtype
-;	  CHANNEL_START = 1
-;	  CHANNEL_END = 2
-;	  HANGUP = 3
-;	  ANSWER = 4
-;	  APP_START = 5
-;	  APP_END = 6
-;	  BRIDGE_START = 7
-;	  BRIDGE_END = 8
-;	  CONF_START = 9
-;	  CONF_END = 10
-;	  PARK_START = 11
-;	  PARK_END = 12
-;	  BLINDTRANSFER = 13
-;	  ATTENDEDTRANSFER = 14
-;	  TRANSFER = 15
-;	  HOOKFLASH = 16
-;	  3WAY_START = 17
-;	  3WAY_END = 18
-;	  CONF_ENTER = 19
-;	  CONF_EXIT = 20
-;	  USER_DEFINED = 21
-;	  LINKEDID_END = 22
-;	  BRIDGE_UPDATE = 23
-;	  PICKUP = 24
-;	  FORWARD = 25
-;	eventtime  (timeval, includes microseconds)
-;	userdeftype (set only if eventtype == USER_DEFINED)
-;	cid_name
-;	cid_num
-;	cid_ani
-;	cid_rdnis
-;	cid_dnid
-;	exten
-;	context
-;	channame
-;	appname
-;	appdata
-;	accountcode
-;	peeraccount
-;	uniqueid
-;	linkedid
-;	amaflags  (an int)
-;	userfield
-;	peer
-;	extra
+;   eventtype
```

Comment:
Was the change in formatting intentional or an artifact of your editor saving the file?

Reply:
It was intentional to follow the spacing of the other files.

I can revert back.

Reply:
Nope that's fine.